src/components/FormRegister.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <form class="form-wrapper" @submit.prevent="onSubmit">
+    <BaseInput
+      type="email"
+      label="email"
+      placeholder="Enter your email address"
+      v-model="email"
+      @blur="isEmailTouched = true"
+    />
+    <div>{{ emailError }}</div>
+    <PasswordInput
+      label="password"
+      v-model="password"
+      @blur="isPasswordTouched = true"
+    />
+    <div>{{ passwordError }}</div>
+    <div class="form-wrapper__hints">
+        <p>At least 8 characters</p>
+        <p>At least one letter</p>
+        <p>At least one digit</p>
+    </div>
+    <div class="form-wrapper__action">
+       <router-link :to="{ name: 'Login' }">
+      <BaseButton caption="Log in instead" buttonType="secondary"/>
+     </router-link>
+      <BaseButton caption="Next step" type="submit"/>
+    </div>
+   
+  </form>
+</template>
+
+<script>
+/* eslint-disable */
```

Comment:
if you are tired by eslint errors you should probably change the config or some rules that are annoying, instead of disabling it for the file :) 

Hint:
Did you know that in VSCode you can ask ESLint to fix your file according to it's suggestions?

Assuming that you have ESLint extension installed: click `cmd+shift+P` and start typing the command: "fix all" then select:
```
ESLint: Fix all auto-fixable Problems
```

Reply:
Thanks, I will try do sth with that.